fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -467,7 +468,8 @@ protected static void addCacheControlHeaders(final HttpServletResponse servletRe
         final Date date = mutableResource.getLastModifiedDate();
 
         if (!etag.getValue().isEmpty()) {
-            servletResponse.addHeader("ETag", etag.toString());
+            servletResponse.addHeader("ETag", mutableResource.hasType(FEDORA_NON_RDF_SOURCE_DESCRIPTION) ?
+               etag.toString() : "W/" + etag.toString());
```

Comment:
No need for `"W/" + etag.toString())` because `toString()` is implicit. Just `"W/" + etag` is fine. In fact I would prefer:
```
servletResponse.addHeader("ETag", mutableResource.hasType((FEDORA_NON_RDF_SOURCE_DESCRIPTION) ?
+              "" : "W/" ) + etag);
```